# crtbeginS.o and crtendS.o come with GCC and are not at their default locations when
# building with easybuild. So we allow a prefix.
# by wpoely86@gmail.com
diff -urN llvm-3.4.orig/tools/clang/test/Driver/pic.c llvm-3.4/tools/clang/test/Driver/pic.c
--- llvm-3.4.orig/tools/clang/test/Driver/pic.c	2013-10-04 13:46:54.000000000 +0200
+++ llvm-3.4/tools/clang/test/Driver/pic.c	2014-04-25 11:23:30.772175180 +0200
@@ -21,8 +21,8 @@
 //
 // CHECK-PIE-LD: "{{.*}}ld{{(.exe)?}}"
 // CHECK-PIE-LD: "-pie"
-// CHECK-PIE-LD: "Scrt1.o" "crti.o" "crtbeginS.o"
-// CHECK-PIE-LD: "crtendS.o" "crtn.o"
+// CHECK-PIE-LD: "Scrt1.o" "crti.o" "{{.*}}crtbeginS.o"
+// CHECK-PIE-LD: "{{.*}}crtendS.o" "crtn.o"
 //
 // CHECK-NOPIE-LD: "-nopie"
 //
